### PR TITLE
feat(SPV-1018): Fix Dark Mode

### PR DESCRIPTION
**Pull request by Nazarii-4chain**
# Pull Request Checklist

- [ ] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/bitcoin-sv/spv-wallet-admin/blob/main/.github/CODE_STANDARDS.md)
- [ ] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/spv-wallet-admin/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [ ] 📝 I have used conventional commits.
- [ ] 📗 I have updated any related documentation.
- [ ] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

**Review by dzolt-4chain — APPROVED** [review]

File: tailwind.config.js [hunk 1]
```diff
@@ -1,75 +1,77 @@
...
-        'accordion-up': 'accordion-up 0.2s ease-out',
+        "accordion-down": "accordion-down 0.2s ease-out",
+        "accordion-up": "accordion-up 0.2s ease-out",
       },
     },
   },
-  plugins: [animatePlugin],
-};
+  plugins: [require("tailwindcss-animate")],
+}
```
Comment (swierzbicki) [inline tailwind.config.js:77]: No new line in the end of file

File: tailwind.config.js [hunk 2]
```diff
@@ -1,75 +1,77 @@
...
+    './components/**/*.{ts,tsx}',
+    './app/**/*.{ts,tsx}',
+    './src/**/*.{ts,tsx}',
+  ],
+  prefix: "",
   theme: {
     container: {
       center: true,
-      padding: '2rem',
+      padding: "2rem",
```
Comment (swierzbicki) [inline tailwind.config.js:14]: is `' '` caused some issues or what? Why have we changed to `" "` ?

File: eslint.config.js [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
+import reactRefresh from 'eslint-plugin-react-refresh';
+import globals from 'globals';
+import ts from 'typescript-eslint';
 
 export default [
   eslint.configs.recommended,
   ...ts.configs.recommended,
   {
-    ignores: ['dist/**/*', 'node_modules/**/*', 'eslint.config.js'],
+    ignores: ['dist/**/*', 'node_modules/**/*', 'eslint.config.js', 'tailwind.config.js'],
```
Comment (swierzbicki) [inline eslint.config.js:15]: To be honest i don' see anything wrong to format config files with prettier. At least there will be one, consistent approach. 

I would remove albo `'eslint.config.js'`

File: eslint.config.js [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
-import importPlugin from 'eslint-plugin-import';
-import reactPlugin from 'eslint-plugin-react';
 
 import * as tanstackQuery from '@tanstack/eslint-plugin-query';
+import importPlugin from 'eslint-plugin-import';
+import reactPlugin from 'eslint-plugin-react';
+import reactHooks from 'eslint-plugin-react-hooks';
+import reactRefresh from 'eslint-plugin-react-refresh';
+import globals from 'globals';
+import ts from 'typescript-eslint';
```
[the remark this answers is not in the document]
Comment (Nazarii-4chain) [reply eslint.config.js:9]: It's auto ordered by IDE. I can revert this if needed.

File: tailwind.config.js (hunk 2 above)
Comment (Nazarii-4chain) [reply tailwind.config.js:14]: autogenerated by shadcdn. Can revert this if needed

File: eslint.config.js (hunk 3 above)
Comment (Nazarii-4chain) [reply eslint.config.js:15]: Don't quite understand what you mean. These changes related to eslint

File: tailwind.config.js (hunk 1 above)
Comment (Nazarii-4chain) [reply tailwind.config.js:77]: Adde new line

File: tailwind.config.js (hunk 2 above)
Comment (swierzbicki) [reply tailwind.config.js]: Yes. Please. And we could think to maybe include configs into prettier. And work would be done by prettier 🤔 
We will see. Comment marked as Resolved

File: eslint.config.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (swierzbicki) [reply eslint.config.js:9]: Yes please :)

File: eslint.config.js (hunk 3 above)
Comment (swierzbicki) [reply eslint.config.js:15]: My fault. Don't know why but i assumed that this is a prettier config. Comment marked as Resolved

**Review by chris-4chain — APPROVED** [review]

**Pull request merged**